openshift-knative-operator/pkg/common/config_test.go

```diff
@@ -74,3 +74,83 @@ func TestConfigure(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigureIfConfigmapUnset(t *testing.T) {
+	cases := []struct {
+		name     string
+		in       base.ConfigMapData
+		expected base.ConfigMapData
+	}{{
+		name: "all nil",
+		expected: base.ConfigMapData{
+			"foo": map[string]string{
+				"bar": "baz",
+			},
+		},
+	}, {
+		name: "first level already set but empty configuration",
+		in: base.ConfigMapData{
+			"foo": map[string]string{},
+		},
+		expected: base.ConfigMapData{
+			"foo": map[string]string{
+				"bar": "baz",
+			},
+		},
+	}, {
+		name: "override",
```

Comment:
Should not this test be named `do not override if set` or something?
If I understand correctly the goal is to configure only if unset right?